website/static/js/newAndNoteworthyPlugin.js

```diff
@@ -0,0 +1,195 @@
+            location.href = '/' + node.embeds.target_node.data.id;
+        };
+
+        self.redirectToSearch = function() {
+            location.href = '/search';
+        };
+
+        self.addToolTip = function(line) {
+            var $line = $(line);
+            console.log(line.offsetWidth, line.scrollWidth);
+            if (line.offsetWidth < line.scrollWidth && !$line.attr('title')) {
+                $line.attr('title', $line.text());
+            }
+        };
+    },
+    view : function(ctrl) {
+        function nodeDisplay(node) {
+            return m('div', {'class': 'row node-styling noteworthy-spacing', onclick: function(){{ctrl.nodeDirect(node);}}},
+                m('div', {'class': 'col-sm-12'},
+                    m('h5', {'class': 'prevent-overflow', onmouseover: function(){ctrl.addToolTip(this);}}, m('em', ctrl.getTitle(node))),
+                    m('h5', {'class': 'prevent-overflow', onmouseover: function(){ctrl.addToolTip(this);}},  ctrl.getDescription(node)),
+                    m('h5', {'class': 'prevent-overflow'}, m('span', {'class': 'f-w-xl'}, 'Contributors: '), m('span',  ctrl.getContributors(node)))
+                )
+            );
+        }
+
+        function populateNodesSmallScreen () {
+            var formattedNodes = [];
+            formattedNodes.push(
+                m('div', {'class': 'row sm-new-and-noteworthy'}, m('div', {'class':'col-sm-10 col-sm-offset-1'},
+                    m('div', {'class': 'col-sm-6'}, m('h4', 'New and Noteworthy')),
+                    m('div', {'class': 'col-sm-6'}, m('h4', 'Most Popular')))
+            ));
+            for (var i = 0; i <= ctrl.popularNodes().length - 1; i++) {
+                var newNoteworthy = ctrl.newAndNoteworthyNodes()[i];
+                var popular = ctrl.popularNodes()[i];
+                formattedNodes.push(m('div', {'class': 'row sm-new-and-noteworthy'},
+                    m('div', {'class': 'col-sm-10 col-sm-offset-1'},
+                        m('div', {'class': 'col-sm-6'}, nodeDisplay(newNoteworthy)),
+                        m('div', {'class': 'col-sm-6'}, nodeDisplay(popular))
+                    )
+                ));
+            }
+            return formattedNodes;
+        }
+
+        function newAndNoteworthyProjectsTemplateXSScreen () {
+            return ctrl.newAndNoteworthyNodes().map(function(node){
+                return nodeDisplay(node);
+            });
+        }
+
+        function popularProjectsTemplateXSScreen () {
+            return ctrl.popularNodes().map(function(node){
+                return nodeDisplay(node);
+            });
+        }
+
+        function findMoreProjectsButton () {
+            return m('button', {type:'button', 'class':'btn btn-default m-v-lg', onclick: function(){
+                ctrl.redirectToSearch();
+            }}, 'Find more projects with advanced search');
+        }
+
+        return m('div', {'class': 'container'}, [
```

Comment:
Left in - quick search, new and noteworthy, and meetings and conferences are individually wrapped in containers.